core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -858,39 +861,45 @@ public boolean load() {
 
         if (!allowWrites && dataAccessDefaultType.isMMap())
             dataAccessDefaultType = DAType.MMAP_RO;
-        if (encodingManager == null)
-            // we did not call init(), so we build the encoding manager now.
-            // just like when calling init, users have to make sure they use the same setup for import and load
-            buildEncodingManagerAndOSMParsers(flagEncodersString, encodedValuesString, dateRangeParserString, profilesByName.values());
 
```

Comment:
This problem no longer exists as we do not build the encoding manager in `init()` anymore and it does not matter whether we called it or not.